src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleNoParens.java

```diff
@@ -36,10 +36,11 @@ enum SON3 {
     ETHAN
 }
 
-@InputAnnotationUseStyleCustomAnnotation6() // violation
+@InputAnnotationUseStyleCustomAnnotation6// violation 'Annotation cannot have closing parenthesis'
+             ()
```

Comment:
Keep code same, put comment above

Reply:
Missed this line by mistake. Respective changes to be made are in progress.

Reply:
Done.